hazelcast-sql/pom.xml

```diff
@@ -720,6 +720,11 @@
                     <groupId>org.apache.yetus</groupId>
                     <artifactId>audience-annotations</artifactId>
                 </exclusion>
+                <!-- Exclusions for Zookeeper. -->
```

Comment:
no `Uzaygezen` in master?

Reply:
I guess Calcite maintainers finally excluded `log4j:1.2.17` from `calcite-core`.